pkg/infra/iac2/inputs_outputs_test.go

```diff
@@ -150,23 +151,40 @@ func TestKnownTemplates(t *testing.T) {
 					t.Run(inputName, func(t *testing.T) {
 						assert := assert.New(t)
 
-						field, fieldFound := resType.FieldByName(inputName)
+						var inputType reflect.Type
 
-						if !assert.Truef(fieldFound, `missing field`, field.Name) {
+						if field, fieldFound := resType.FieldByName(inputName); fieldFound {
```

Comment:
Is this part of the refactor, or unrelated additional testing?

Reply:
Yes because the `Name` field has been removed on some resources that delegate their name to another resource (VpcLink for eg).